lib/constants.js

```diff
@@ -98,11 +98,11 @@ module.exports = {
   REGEX_REMOVE_BACKSLASH: /(?:\[.*?[^\\]\]|\\(?=.))/g,
 
   // Replace globs with equivalent patterns to reduce parsing time.
-  REPLACEMENTS: {
+  REPLACEMENTS: Object.assign(Object.create(null), {
     '***': '*',
     '**/**': '**',
     '**/**/**': '**'
-  },
+  }),
```

Comment:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#null-prototype_objects), this can also be achieved with the object literal syntax:
```js
REPLACEMENTS: {
  __proto__: null,
  '***': '*',
  '**/**': '**',
  '**/**/**': '**'
}
```

This seems like it would be faster because it only creates one object, but I haven't benchmarked it.

Reply:
Object assign is a bit confusing. I didn't realize that `.create` took an initializer.